logs/repl/README.md

```diff
@@ -1,4 +1,4 @@
-# repl directory
+# Repl directory
```

Comment:
The directory is named `repl`, not `Repl`. Directory names are case sensitive in many operating systems, so this is relevant.

Perhaps it'd be clearer as <code># `logs/repl/` directory</code>